charts/gitops-runtime/templates/_components/cap-app-proxy/_service.yaml

```diff
@@ -11,7 +11,11 @@ spec:
     - port: {{ .Values.service.port }}
       targetPort: http
       protocol: TCP
-      name: http
+      name: htt
+    - port: 9100
+      targetPort: http
+      protocol: TCP
+      name: http-metrics
```

Comment:
temporarily here, to be able to scrape metrics and check memory usage. not needed to be merged. wouldn't be the end of the world if it does get merged.